### PR TITLE
Nothing makes sense

### PR DESCRIPTION
**Pull request by kythant**

### CODE REVIEW

**Comment by kythant** [comment]
image
Going to just live with this pipeline triggering two builds

File: build/ProjectReunion-BuildFoundation.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ parameters:
     default: False
   - name: "IgnoreFailures"
     type: boolean
-    default: False
+    default: True
```
Comment (DrusTheAxe) [inline build/ProjectReunion-BuildFoundation.yml:12]: Why?

Comment (kythant) [reply build/ProjectReunion-BuildFoundation.yml:12]: Was testing the build validation policies. Not meant to be a real PR

**Pull request closed without merging**